benchs/bench_ivfpq_cuvs.py

```diff
@@ -41,13 +41,13 @@ def aa(*args, **kwargs):
 
 
 group = parser.add_argument_group('benchmarking options')
-aa('--raft_only', default=False, action='store_true',
```

Comment:
Have you run this before and after the change to determine the change (if any) in performance?